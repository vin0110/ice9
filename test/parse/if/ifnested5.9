if a ->
  if b ->
    if c ->
      if d ->
	if e ->
	  f();
	[] g ->
	  h();
	fi
      [] else ->
	i();
      fi
    [] j -> k();
    fi
  [] else -> l();
  fi
[] m ->
  n;
fi
