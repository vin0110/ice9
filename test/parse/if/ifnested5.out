IF:
  ID:'a'
(then)
  IF:
    ID:'b'
  (then)
    IF:
      ID:'c'
    (then)
      IF:
        ID:'d'
      (then)
        IF:
          ID:'e'
        (then)
          CALL:
            ID:'f'
          (arglist)
            nil
        (else)
          IF:
            ID:'g'
          (then)
            CALL:
              ID:'h'
            (arglist)
              nil
          (else)
            nil
      (else)
        CALL:
          ID:'i'
        (arglist)
          nil
    (else)
      IF:
        ID:'j'
      (then)
        CALL:
          ID:'k'
        (arglist)
          nil
      (else)
        nil
  (else)
    CALL:
      ID:'l'
    (arglist)
      nil
(else)
  IF:
    ID:'m'
  (then)
    ID:'n'
  (else)
    nil
